test/contract-browser/chromium.contract.test.ts

```
@@ -212,6 +249,128 @@ function createObservedLauncher(observation: MutableOperationObservation): Playw
+    });
+    probeHandle = await probeLocator.elementHandle();
+    locatorPrototype = Object.getPrototypeOf(probeLocator) as MutableLocatorPrototype;
+    physicalHandlePrototype = Object.getPrototypeOf(probeHandle) as MutablePhysicalHandlePrototype;
+  } finally {
+    const cleanup = await Promise.allSettled([
+      probeHandle?.dispose(),
+      probeBrowser.close(),
+    ]);
+    const failure = cleanup.find((result) => result.status === 'rejected');
+    if (failure?.status === 'rejected') {
+      throw failure.reason;
```

Biome (2.5.6)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)